plugins/inputs/openweathermap/README.md

```diff
@@ -0,0 +1,67 @@
+# Telegraf Plugin: openweathermap
+
+OpenWeatherMap provides the current weather and forecasts for more than 200,000 cities. To use this plugin you will need a token. For more information [click here](https://openweathermap.org/appid).
+
+Find city identifiers in this [list](http://bulk.openweathermap.org/sample/city.list.json.gz). You can also use this [url](https://openweathermap.org/find) as an alternative to downloading a file. The ID is in the url of the city: `https://openweathermap.org/city/2643743`
+
+### Configuration:
+
+```toml
+[[inputs.openweathermap]]
+  ## Root url of API to pull stats
+  # base_url = "http://api.openweathermap.org/data/2.5/"
+  ## Your personal user token from openweathermap.org
+  # app_id = "xxxxxxxxxxxxxxxxxxxxxxx"
+  ## List of city identifiers
+  # cities = ["2988507", "519188"]
+  ## HTTP response timeout (default: 5s)
+  # response_timeout = "5s"
+  ## Query the next ten days forecast (default: true)
+  # forecast_enable = true
+```
+
+### Metrics:
+
++ weather
+  - fields:
+    - humidity (int, Humidity percentage)
+    - temperature (float, Unit: Celcius)
+    - pressure (float, Atmospheric pressure in hPa)
+    - rain (float, Rain volume for the last 3 hours, mm)
+    - wind_speed (float, Wind speed. Unit Default: meter/sec)
+    - wind_deg (float,  Wind direction, degrees)
+  - tags:
+    - city_id
+    - forecast
+
+### Example Output:
+
+Using this configuration:
+```toml
+[[inputs.openweathermap]]
+  base_url = "http://api.openweathermap.org/data/2.5/"
```

Comment:
Lets use `https` as the default too, seems to work and will hide the users api key a bit better.